setup.py

```diff
@@ -27,13 +27,18 @@
     "appdirs>=1.4.3",
     "Click>=7.0",
     "diskcache>=4.0.0",
-    "inflection>=0.3.1",
     "iso8601>=0.1.12",
     "Jinja2>=2.10.1,<3.0",
     "jsonschema>=3.0.2",
+    # 'markupsafe' is required by Jinja2. From version 2.0.0 on
+    # py3.5 support is dropped.
+    "markupsafe<2.0.0",
```

Comment:
Just to be safe, lets add `markupsafe>=1.1,<2.0.0` so we specify the minimum version of this required by the version of Jinja2 we are using.  The resolver *should* work that out on its own, but in my experience `pip` doesn't always do a great job of that, so more hints is usually better.

Reply:
Great point!